sphinx/locale/__init__.py

```diff
@@ -242,6 +242,16 @@ def _(message):
             return message
 
 
+def __(message):
+    # type: (unicode) -> unicode
+    """A dummy wrapper to i18n'ize exceptions and command line messages.
+
+    In future, the messages are translated using LC_MESSAGES or any other
+    locale settings.
```

Comment:
At present, `sphinx.locale` can't translate multiple languages at same time.
So I made this as pass-through function.

In future version, I will implement this function to i18n'ize command line.